scripts/kind-e2e/e2e.sh

```diff
@@ -122,7 +122,7 @@ function create_subm_vars() {
   ce_ipsec_ikeport=500
   ce_ipsec_nattport=4500
 
-  subm_ns=operators
+  subm_ns=submariner-operator
```

Comment:
Should we make other OLM/docs changes to match this namespace?

Reply:
Good question. With OLM you tell the olm where to put it, in my docs I picked "openshift-submariner".  We could standardize to just submariner-operator may be, so it's uniform u/s and d/s.

If we give freedom to pick one subctl could get confused if you provide a different ns, because it could look like the operator is not installed, although it is.. (in the default namespace..).